packit.yaml

```diff
@@ -18,3 +18,13 @@ jobs:
       - fedora-35
       - centos-stream-8-x86_64
       - centos-stream-9-x86_64
+
+  # run build/unit tests on some interesting architectures
+  - job: copr_build
+    trigger: pull_request
+    metadata:
+      targets:
+        # 32 bit
+        - fedora-development-i386
```

Comment:
:thinking: Hmm, should we use fedora-stable-* rather? Rawhide is more likely to fail and thus catch relevant problems earlier -- at the same time exactly that can also be annoying. I just wouldn't want to waste *too* many cycles on building everywhere.

Reply:
Perhaps, let's start with rawhide, and once it gets in the way we have a look and consider switching to stable?

Reply:
Sure, it is fine to use rawhide unless it becomes too broken too often.
Don't we want to also run integration tests?

Reply:
s390x (and 32 bit arm) are emulated architectures, running browser tests is way too slow there. We could in principle run them on i386, not sure how well browsers are actually supported there. Fedora doesn't support 32 bit desktop any more, but let's try in #16876 just to get the data.

Also, I think unit tests should be enough -- JavaScript isn't really architecture sensitive. We could of course discover 32 bit bugs in the Linux stack.

Reply:
Running testing-farm tests on i386 [is not a thing](https://github.com/cockpit-project/cockpit/pull/16876/checks?check_run_id=4934762122), so I closed #16876.